refresh documentation (#693)
nitrictech/docs: docs/guides/nodejs/byo-database.mdx

  - typescript
  - javascript
published_at: 2022-10-13
updated_at: 2024-12-30
---

# BYO Database

Nitric currently has out of the box **preview** support for [PostgreSQL databases](/sql), however Nitric allows you to use whatever tooling and ORMs you prefer for directly interfacing with your database. If you want to use a database that isn't managed by Nitric, we have the following recommendations:

> - For local development is to set up a container that runs alongside your Nitric processes.

Reviewer: ```suggestion
- For local development, set up a container that runs alongside your Nitric processes.
```

> - For a production environment, you can use any of the database services for your preferred cloud:

Reviewer: ```suggestion
- For a production environment, use any of the database services offered by your preferred cloud provider:
```
